### PR TITLE
Misc minor fixes

### PR DESCRIPTION
**Pull request by hesingh**
Fixes:

https://github.com/p4lang/p4c/issues/2135

https://github.com/p4lang/p4c/issues/2117

and a minor spelling mistake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ versions in the 3 series may not be supported by other p4lang projects, such as
 well, but all our CI testing is done with version 3.2.0. After cloning protobuf
 and before you build, check-out version 3.2.0:
 
-`git checkout v3.2.0`
+`git checkout v3.6.1`
```
Comment (jafingerhut) [inline README.md]: There are multiple occurrences of the version number 3.2.0 in the README.  It would be very confusing only to update this one.

Comment (hesingh) [reply README.md]: Since p4c is now using protobuf 3.6.1, would folks mind if I changed other instances of 3.2.0  to 3.6.1?  thanks.hh

Comment (hesingh) [reply README.md]: I just verified p4lang/PI and even this repo has upgraded to using protobug 3.6.1.  See

https://github.com/p4lang/PI/commit/1539ecd8a50c159b011d9c5a9c0eba99f122a845

Comment (jafingerhut) [reply README.md]: As far as I know, everything still works with protobuf 3.2.0, too.  @antoninbas Any opinion on what the p4c README should say about protobuf versions?  In particular, OK to replace all mentions of 3.2.0 with 3.6.1?  It doesn't seem critical to make such a change if both work, but if other open source P4 tools are "officially supporting" version 3.6.1, it seems to make sense to update all relevant README/install scripts eventually.

**Comment by hesingh** [comment]
I don't know about Bazel being used to build any p4runtime subset of p4c, but the URL in the following file is incorrect.  https://github.com/p4lang/protobuf/archive does not exist.

control-plane/p4runtime/bazel/rules.bzl:            "url": "https://github.com/p4lang/protobuf/archive/v3.2.0-patched.zip",

File: README.md (hunk 1 above)
Comment (hesingh) [reply README.md]: One may note that p4lang/PI documentation has already moved to using protobuf 3.6.1.

**Comment by antoninbas** [comment]
We should use 3.6.1 consistently for the version. This is the version we used in the docker images in which we run CI.

**Comment by hesingh** [comment]
If there aren't any more comments, anyone will merge permission, please merge this PR - thanks.

**Comment by hesingh** [comment]
thanks!

**Pull request merged**